site/docs/doc2.md

```diff
@@ -1,6 +1,27 @@
 ---
 id: doc2
-title: Document Number 2
+title: Run your own Raiden Node
 ---
 
-This is a link to [another document.](doc3.md) This is a link to an [external page.](http://www.example.com/)
+## Setup Node
```

Comment:
@wynstrol Is it possible to add the link to the repo, the clone instructions, etc. I think its best to write this as if you were giving instructions to another developer that had 'fokol' idea of what even a node was, or why it's even needed